libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1662,7 +1662,7 @@ void AP_GPS::handle_gps_rtcm_fragment(uint8_t flags, const uint8_t *data, uint8_
     if (rtcm_buffer->fragment_count != 0 &&
         rtcm_buffer->fragments_received == (1U << rtcm_buffer->fragment_count) - 1) {
         // we have them all, inject
-        rtcm_stats.fragments_used += rtcm_buffer->fragments_received;
+        rtcm_stats.fragments_used += __builtin_popcount(rtcm_buffer->fragments_received);
```

Comment:
```suggestion
        rtcm_stats.fragments_used += rtcm_buffer->fragment_count;
```

We just established these were equal.

Reply:
8 != 256, could use fragment_count as well but I think the current fix is technically correct